linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/StatusCodeStatsFilter.scala

```diff
@@ -15,7 +16,55 @@ object StatusCodeStatsFilter {
       def make(_stats: param.Stats, next: ServiceFactory[Request, Response]) = {
         val param.Stats(statsReceiver) = _stats
         if (statsReceiver.isNull) next
-        else new StatsFilter[Request](statsReceiver) andThen next
+        else new StatusCodeStatsFilter(statsReceiver).andThen(next)
       }
     }
 }
+
+/**
+ * Modified from com.twitter.finagle.http.filter.StatsFilter.
```

Comment:
That is intentional, since response size computation is totally broken for chunked responses, which is really misleading. We should track _request_ and response body sizes and durations, but I'm happy to drop them for the time being.  Should ticket the fix, though.